components/script/dom/htmlmediaelement.rs

```diff
@@ -404,6 +404,8 @@ pub enum NetworkState {
 /// <https://html.spec.whatwg.org/multipage/#dom-media-readystate>
 #[derive(Clone, Copy, Debug, JSTraceable, MallocSizeOf, PartialEq, PartialOrd)]
 #[repr(u8)]
+#[allow(clippy::enum_variant_names)]
+/// Clippy warning silenced here because these names are from the specification.
```

Comment:
This needs to be normal comment `//` not rustdoc comment `///`, then rustfmt should not complain.
```suggestion
#[allow(clippy::enum_variant_names)] // Clippy warning silenced here because these names are from the specification.
```

Reply:
Its alright, I just changed the formatting. It works now and no erros are detected by ./mach build --dev or ./mach test-tidy

Reply:
Rustdoc comments are used to write docs, while normal comments are used for anything else. As this is a comment that does not document stuff, but only explains why allow is needed it needs to be changed.